tests/crypto/crl_tests.c

```diff
@@ -139,22 +152,17 @@ static void _test_get_dates(void)
     oe_datetime_t next;
     OE_TEST(oe_crl_get_update_dates(&crl, &last, &next) == OE_OK);
 
-    OE_TEST(last.year == _time.year);
-    OE_TEST(last.month == _time.month);
-    OE_TEST(last.day == _time.day);
-    OE_TEST(last.hours == _time.hours);
-    OE_TEST(last.minutes == _time.minutes);
-    // TEMPORARILY Disabled: _time is the certificate file's timestamp
-    // whereas last.seconds is the actual issue time of the certificate.
-    // The two values are not guaranteed to be equal.
-    // OE_TEST(last.seconds == _time.seconds);
+    int32_t diff = _diff_oe_datetime(&last, &_time);
+    printf("=== Diff between last and current time: %d\n", diff);
+    OE_TEST(abs(diff) < 60); // Ensure times are within 60 seconds
```

Comment:
If the tolerance needs to in minutes can we instead just add the tolerance to the check that compares the minutes?:

`OE_TEST( abs((int32_t)last.minutes -_(int32_t)time.minutes) <= 1)`;

Reply:
minute 1, second 59
minute 2, second 1

How far apart are these?

Reply:
Tolerance is in seconds. 60 was a random choice, a much smaller number would be ideal